microservices-connector/cmd/router/main.go

```diff
@@ -517,7 +611,14 @@ func routeStep(nodeName string,
 }
 
 func mcGraphHandler(w http.ResponseWriter, req *http.Request) {
-	ctx, cancel := context.WithTimeout(req.Context(), time.Minute)
+
+	// TODO: traces
+	// span := trace.SpanFromContext(ctx)
+	// bag := baggage.FromContext(ctx)
+	// span.AddEvent("handling this...", trace.WithAttributes(uk.String(bag.Member("username").Value())))
+
+	_, _ = io.WriteString(w, "Hello, world!\n")
+	ctx, cancel := context.WithTimeout(req.Context(), 3600*time.Minute)
```

Comment:
Hi @ppalucki,  may 1 hour time out be too long?

Reply:
@zhlsunshine fixed (long timeout was used it for my debugging sessions)